### ISSUE TITLE
 uninitialized constant Sprockets::Helpers::AssetNotPrecompiledError

### ISSUE THREAD

**Issue by aquach** [ndbroadbent/turbo-sprockets-rails3#15]
```
NameError: uninitialized constant Sprockets::Helpers::AssetNotPrecompiledError
/usr/local/html/member/vendor/bundle/ruby/1.8/gems/turbo-sprockets-rails3-0.1.13/lib/turbo-sprockets/sprockets_overrides/helpers/rails_helper.rb:37:in `const_missing'
/usr/local/html/member/vendor/bundle/ruby/1.8/gems/turbo-sprockets-rails3-0.1.13/lib/turbo-sprockets/sprockets_overrides/helpers/rails_helper.rb:37:in `digest_for'
/usr/local/html/member/vendor/bundle/ruby/1.8/gems/actionpack-3.2.3/lib/sprockets/helpers/rails_helper.rb:150:in `rewrite_asset_path'
```

**Comment by ndbroadbent** [comment]
Hi there, thanks for the bug report! I've fixed this, so please try updating to `0.1.14` and try again.

However, it looks like you are still going to receive a `AssetNotPrecompiledError` error on your next try.. I will look into that

**Comment by ndbroadbent** [comment]
Closing as duplicate of issue #14. I would be grateful if you could provide more information by following the instructions in this comment: https://github.com/ndbroadbent/turbo-sprockets-rails3/issues/14#issuecomment-9326126
